startbootstrap-sb-admin-gh-pages/tables.html

```diff
@@ -673,6 +210,65 @@ <h5 class="modal-title" id="exampleModalLabel">Ready to Leave?</h5>
     </div>
   </div>
 
+  <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js'></script>
+  <script type="text/javascript">
+    function refresh(){
+      let xhr = new XMLHttpRequest();
+      var url = "http://localhost:5002/all";
+      xhr.open('GET', url)
+      xhr.send()
+      xhr.onreadystatechange = function() {
+        if (xhr.readyState==4 && xhr.status==200) {
+          var data = JSON.parse(xhr.responseText);
+          console.log(data.length)
+
+          // tableRef = document.getElementById('table')
+          // var newRow   = tableRef.insertRow();
+          // // Insert a cell in the row at index 0
+          // var cell0 = newRow.insertCell();
+          // var cell1 = newRow.insertCell();
+          // var cell2  = newRow.insertCell();
+          // var cell3  = newRow.insertCell();
+          // var cell4  = newRow.insertCell();
+          // var cell5  = newRow.insertCell();
+          // var cell6  = newRow.insertCell();
+          // var cell7  = newRow.insertCell();
+          for (i = 0; i< data.length; i++){
+            tableRef = document.getElementById('table')
+            var newRow   = tableRef.insertRow();
+          // Insert a cell in the row at index 0
+            var cell0 = newRow.insertCell();
+            var cell1 = newRow.insertCell();
+            var cell2  = newRow.insertCell();
+            var cell3  = newRow.insertCell();
+            var cell4  = newRow.insertCell();
+            var cell5  = newRow.insertCell();
+            var cell6  = newRow.insertCell();
+            var cell7  = newRow.insertCell();
+            var co  = document.createTextNode(data[i].cash_out),
+            bi = document.createTextNode(data[i].buy_in),
+            loc = document.createTextNode(data[i].location)
+            bb = document.createTextNode(data[i].big_blind)
+            lb = document.createTextNode(data[i].little_blind)
+            loc = document.createTextNode(data[i].location)
+            ng = document.createTextNode(data[i].net_gain)
+            st =  document.createTextNode(data[i].startime)
+            et = document.createTextNode(data[i].endtime)
+
+              cell0.appendChild(bi);
+              cell1.appendChild(co);
+              cell2.appendChild(ng);
+              cell3.appendChild(loc);
+              cell4.appendChild(lb);
+              cell5.appendChild(bb);
+              cell6.appendChild(st);
+              cell7.appendChild(et);
+            }
```

Comment:
this should end on the same indentation that it started. I think its tabbed one too many.